packages/flutter_adaptive_scaffold/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: flutter_adaptive_scaffold
 description: Widgets to easily build adaptive layouts, including navigation elements.
-version: 0.0.4
+version: 0.0.5
```

Comment:
Should be a non-breaking change because `AppBar` implements `PreferredSizeWidget`.

Reply:
I'm leaning more toward a minor version change versus a patch version change. 

> Patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.
> ...
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

https://semver.org/

cc @gspencergoog

Reply:
Sounds fine to me.

Reply:
Changed to `0.1.0`

Reply:
0.0.4->0.1.0 is a breaking change in Dart; see the second-to-last paragraph in https://dart.dev/tools/pub/versioning#semantic-versions. While it describes it as a "convention", it's one that's enforced by the `pub` tool; `^0.0.x` will not allow an update to `0.1.0`.

0.0.4->0.0.5 is a minor version change in Dart; the patch version would be 0.0.4+1.

Reply:
Reverted my change, version is `0.0.5` again.

Reply:
Oh, yes, Stuart is right: if it's not actually a breaking change, we shouldn't update the minor version. Although, this is a brand new package, so it's unlikely that anyone has it pinned to `^0.0.x` yet (because it's unlikely that anyone is using it yet).